### DISCUSSION TITLE
添加onedrive失败：客户端不存在或未为消费者启用

### DISCUSSION THREAD

**Discussion by ventusoon** [AlistGo/alist#388] [Q&A] [answered]
到最后一步获取刷新令牌时提示：我们
无法完成你的请求
未授权客户端：客户端不存在或未为消费者启用。如果您是应用程序开发人员，请通过 Azure 门户中的应用程序注册配置新应用程序，网址为 https://go.microsoft.com/fwlink/?linkid=2083908。
![微信截图_20220118212801](https://user-images.githubusercontent.com/14863512/149946331-eaa19210-b697-4263-be1c-0086053997d1.png)

**Comment by xhofe** [comment, marked as answer]
这不是写的很清楚了吗
